docs/my-app-with-namespace-from-folders/translations/components/de-de.yml

```diff
@@ -0,0 +1 @@
+title: Komponenten
```

Comment:
When these "in-between" translation files are added, existing assertions in `tests/acceptance/index-test.ts` fail.